### PR TITLE
[libbase58] Add new port

### PR DESCRIPTION
**Pull request by NancyLi1013**
`libbase58 `is a C library for Bitcoin's base58 encoding.
It currently only supports Linux platform.

Related issue #7898

No feature need to test.

### CODE REVIEW

**Review by JackBoosY — APPROVED** [review]

File: ports/libbase58/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    AUTOCONFIG
+    NO_DEBUG
+)
+
+vcpkg_install_make()
+
+vcpkg_copy_pdbs()
+
+# Handle copyright
+file(INSTALL ${SOURCE_PATH}/COPYING DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT}/copyright)
```
Comment (JackBoosY) [inline ports/libbase58/portfile.cmake]: should be `file(INSTALL ${SOURCE_PATH}/COPYING DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)`

**Review by JackBoosY — CHANGES_REQUESTED** [review]

**Review by JackBoosY — APPROVED** [review]

**Review by dan-shaw — APPROVED** [review]

**Comment by ras0219-msft** [comment]
Please modify ci.baseline.txt to account for the windows failures

**Comment by NancyLi1013** [comment]
Please feel free to reopen this PR if you need it.

**Pull request closed without merging**